includes/alerts.inc.php

```diff
@@ -281,5 +281,18 @@ function GetContacts($results) {
             $contacts[$user['email']] = $user['realname'];
         }
     }
-    return $contacts;
+
+    $tmp_contacts = array();
+    foreach ($contacts as $email => $name) {
+        if (strstr($email, ',')) {
+            $split_contacts = preg_split("/[,\s]+/", $email);
```

Comment:
Accidental leading or or trailing whitespace will cause empty entries in $split_contacts.  Probably should trim() or check for empty emails.

Or maybe it doesn't matter?